### DISCUSSION TITLE
Cant update to KorGE 5.4.0

### DISCUSSION THREAD

**Discussion by meoowe** [korlibs/korge#2214] [Q&A] [answered]
I saw that v5.4.0 was the newest, so I changed the second line of my libs.versions.toml from` korge = { id = "com.soywiz.korge", version = "5.3.2" }` to `korge = { id = "com.soywiz.korge", version = "5.4.0" }`. When I tried to sync, I got an error in settings.gradle.kts saying failed to apply plugin 'com.soywiz.korge.settings'. Did I miss something else I needed to change?

**Comment by soywiz** [comment]
I have created a PR bumping version of the hello-world: https://github.com/korlibs/korge-hello-world/pull/31
Please, check that the `settings.gradle.kts`, `build.gradle.kts` and `gradle/libs.versions.toml`  matches the ones in that repo

For example: https://github.com/korlibs/korge-hello-world/blob/main/settings.gradle.kts

Because bumped seemed to work

**Reply by 0megaD** [reply, marked as answer]
@soywiz I had the same issue.

The actual problem however was not being unable to apply the plugin:

`Failed to apply plugin 'com.soywiz.korge.settings'.
> Can't find ref '#a54e3b2cacf24e0a0eb87e2580f69f8c81d083ce#v0.0.5' in Repository[/Users/daniel/.kproject/clones/github.com/korlibs/korge-tiled/__git__/.git]`

The problem for me was that `deps.kproject.yml` contained a hash as part of the versioning for some dependencies. This hash doesn't break in 5.3.x but breaks in 5.4.0

So instead of (example for Tiled dependency):
`- https://github.com/korlibs/korge-tiled/tree/v0.0.5/korge-tiled##a54e3b2cacf24e0a0eb87e2580f69f8c81d083ce`
I changed it to
`- https://github.com/korlibs/korge-tiled/tree/v0.0.5/korge-tiled`

**Reply by soywiz** [reply]
Woops. Is that then a regression? Eventually we will use amper directly, but for now it is kproject. I'm not sure why did that broken since I didn't change anything related to that AFAIK in 5.4.0
